project_issue_task_sync/models/project_task.py

```diff
@@ -0,0 +1,77 @@
+# -*- coding: utf-8 -*-
+# © 2017 Therp BV <http://therp.nl>
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+from openerp import api, fields, models
+
+
+class ProjectTask(models.Model):
+    _inherit = 'project.task'
+
+    sync_tasks_issues = fields.Boolean(
+        related=['project_id', 'sync_tasks_issues']
+    )
+    issue_ids = fields.One2many('project.issue', 'task_id', 'Issues')
```

Comment:
AFAICT the base use case is 1 Issue -> 1 Task.
If we want to expand it, IMO it makes more sense to have 1 Issue -> n Tasks, rather than 1 Task -> n Issues...
Could you comment on this?